### ISSUE TITLE
VLAN tag not included frame size and rate limit

### ISSUE THREAD

**Issue by jwbensley** [jwbensley/Etherate#27]
```
[user@ucpe_003 Etherate-master]$ sudo ./etherate -i ens3f2 -v 4093 -M 10000000
...
Physical interface MTU (1514 with headers) is less than
the test frame size (1518 with headers). Test frames shall
be limited to the interface MTU size
...
Frame size is 1514 bytes
...
Seconds         Mbps Tx         MBs Tx          FrmTx/s         Frames Tx
1               10.00           1               833             833
2               10.00           2               833             1666
3               10.00           3               833             2499
```

"Frame size is 1514 bytes" - should say 1518.

Also the MTU warning means that 4 bytes of payload are being chopped from the frame (confirmed with tcpdump).

**Comment by jwbensley** [comment]
Add a note to the documentation about using physical interfaces and not sub-interfaces (for VLANs).

**Comment by jwbensley** [comment]
Fixed in V1.15. Documentation updated.
